db/seeds.d/109-katello-notification-blueprints.rb

```diff
@@ -1,4 +1,10 @@
 blueprints = [
+  {
+    group: N_('Hosts'),
+    name: 'host_lifecycle_expire_soon',
+    message: N_('%{release}: %{number_of_hosts} hosts are approaching end of %{lifecycle} on %{end_date}. Please upgrade them before they expire. Check Report Host - Statuses for detail.'),
```

Comment:
The hosts themselves aren't expiring :)

```suggestion
    message: N_('%{release}: %{number_of_hosts} hosts are approaching end of %{lifecycle} on %{end_date}. Please upgrade them before support expires. Check Report Host - Statuses for detail.'),
```